index.html

```diff
@@ -262,6 +265,53 @@ <h2>
 did:web:example.com%3A3000
         </pre>
     </section>
+    <section>
+      <h2>
+        Key Material and Document Handling
+      </h2>
+      <p>
+        Due to the way most web servers present content, it is likely that a
+        particular did:web document will be served with a media type of
+      <pre>application/json</pre>. If a document is retrieved and it is named
+      <pre>did.json</pre> a few processing rules should apply:
+      <ul>
+        <li>
+          If an
+          <pre>@context</pre> is present at the root of the json document,
+          the document should be processed according to the JSON-LD rules.
+          If this is not possible, or if the document failes processing, the
+          document should be rejected from consideration as a did:web doc.
+        </li>
+        <li>
+          If an
+          <pre>@context</pre> is present at the root of the json document,
+          and is passes JSON-LD processing, and it contains the context
+          <pre>https://www.w3.org/ns/did/v1</pre>, it may be further processed
+          as a
+          DID document as specified by section
+          <a href="https://www.w3.org/TR/did-core/#consumption-0">6.3.2</a> of
+          the
+          [[did-core]] specification.
+        </li>
+        <li>
+          If no
+          <pre>@context</pre> is present, it should be processed via normal
+          JSON rules for DID processing as specified in section
+          <a href="https://www.w3.org/TR/did-core/#consumption">6.2.2</a> of the
+          [[did-core]] specification.
+        </li>
+      </ul>
+      </p>
+      <p>
+        Within a did:web document absolute did URLs must be used whenever
```

Comment:
```suggestion
        Whenever a did URL is present within a did:web document it must be an absolute url
```

I think this reads a bit better.